holidayextras/ui-brand-hx
# Styling DL's in dropdowns

#### What does this PR do? (please provide any background)
- Bootstrap dropdown and design to be like select boxes, each containing a link to elsewhere.
- On Holiday Extras we use one of these dropdowns to display opening times for our call centre, strictly this should maybe be a popover but the bootstrap implementation uses data-attributes which don't suit our formatting.
- Instead I have changed the markup to better represent the content (in pattern library - https://github.com/holidayextras/pattern-library/pull/25) and this PR styles the dropdown to look and behave properly.

#### What tests does this PR have?
None

#### How can this be tested?
Run [Pattern Library](https://github.com/holidayextras/pattern-library) and NPM link it to this PR. You should now see a dropdown like the screenshot below

#### Screenshots / Screencast
<img width="1274" alt="screen shot 2016-01-25 at 17 27 18" src="https://cloud.githubusercontent.com/assets/778942/12558774/614c6e50-c389-11e5-9279-f144b95cf37a.png">


#### What gif best describes how you feel about this work?
![Sweat](https://media.giphy.com/media/5EJHDSPpFhbG0/giphy.gif)

---


#### Reviewers

**Review 1**

**Review 2** \*

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
